lib/device-log/ios-log.js

```diff
@@ -17,6 +16,7 @@ class IOSLog {
     this.udid = opts.udid;
```

Comment:
Where the logger is created?

Reply:
In this driver, it is here: https://github.com/appium/appium-ios-driver/blob/246af060fc491af7762142269b84a9ce3638748a/lib/commands/logging.js#L54-L60

In `appium-xcuitest-driver` it is here: https://github.com/appium/appium-xcuitest-driver/blob/master/lib/commands/log.js#L22-L27

This will necessitate a major bump when published, so the xcuitest driver can be updated.

Reply:
So you mean the `xcodeVersion` property has been already set for iOS driver

Reply:
Yes. It gets set in session creation.